Source/WebKit/UIProcess/Extensions/Cocoa/API/WebExtensionContextAPIScriptingCocoa.mm

```diff
@@ -361,7 +361,7 @@
 
         for (NSString *scriptPath in scriptPaths) {
             NSError *error;
-            if (!extension().resourceStringForPath(scriptPath, &error, WebExtension::CacheResult::No, WebExtension::SuppressNotFoundErrors::Yes)) {
+            if (!protectedExtension()->resourceStringForPath(scriptPath, &error, WebExtension::CacheResult::No, WebExtension::SuppressNotFoundErrors::Yes)) {
```

Comment:
Would be nice to cache in a variable before the loop to avoid repeated calls.

Reply:
Fixed!